README.md

```diff
@@ -10,7 +10,7 @@ This package is based on work from Huy Hoang ([dhhoang](https://github.com/dhhoa
 
 ### Manual read/write
 
-You can create a `reader` that can read from a (input) stream. You can specify text encoding in the `ReaderOptions`, otherwise Utf8 will be used by default.
+You can create a `reader` that can read from a (input) stream. You can specify text encoding in the `ReaderOptions`, otherwise Utf8 will be used by default. There are 4 different `reader`s that you can create - examples are as listed below. 
```

Comment:
There are only 2 different readers, text and binary. The 4 reader objects created below take in different inputs

Reply:
Done.